DOCSP-38517: Go language landing page (#179)
mongodb/docs-landing: source/languages/golang.txt

:template: product-landing
:pl-max-width-paragraphs:
:noprevnext:

===============
MongoDB with Go
===============

.. introduction::
   

>    Build Go applications using MongoDB's :atlas:`Atlas cloud database </>`. Several
>    community-built integrations as well as Open Telemetry support developers
>    looking to build the next generation of Go-based solutions backed by MongoDB.

Reviewer: Hard to parse; "support" initially comes across as a noun, not a verb. also, no space in OpenTelemetry

```suggestion
   Build Go applications using MongoDB's :atlas:`Atlas cloud database </>`. OpenTelemetry
   and community-built integrations support developers
   looking to build the next generation of Go-based solutions backed by MongoDB.
```

   .. button:: Get started with the MongoDB Go Driver
      :uri: https://www.mongodb.com/docs/drivers/go/current/

.. image:: /images/Go.svg
   :alt: Go landing page hero image

Driver
------

Provides a high-level API for working with MongoDB databases consistent with
other MongoDB language drivers.

`Go Driver Documentation → <https://www.mongodb.com/docs/drivers/go/current/>`__

Integrations
------------

> `Open Telemetry → <https://opentelemetry.io/>`__

Reviewer: ```suggestion
`OpenTelemetry → <https://opentelemetry.io/>`__
```
